pom.xml

```diff
@@ -112,7 +114,8 @@
         <version.org.jboss.spec.javax.json.jboss-json-api_1.0_spec>1.0.0.Final</version.org.jboss.spec.javax.json.jboss-json-api_1.0_spec>
         <version.org.jboss.spec.javax.servlet.jboss-servlet-api_3.0_spec>1.0.2.Final</version.org.jboss.spec.javax.servlet.jboss-servlet-api_3.0_spec>
         <version.org.jboss.spec.javax.ws.rs.jboss-jaxrs-api_2.0_spec>1.0.0.Final</version.org.jboss.spec.javax.ws.rs.jboss-jaxrs-api_2.0_spec>
-        <version.org.jboss.spec.web.7.0.api>1.1.0.Final</version.org.jboss.spec.web.7.0.api>
+        <version.org.jboss.spec.web.7.0.api>1.1.1.Final</version.org.jboss.spec.web.7.0.api>
```

Comment:
Why did you move back to 1.1.1.Final? See https://github.com/Ladicek/swarm-test-suite/commit/871cdc990f7f63852368c7312c0622712f7959fd

Reply:
Oh f**k I was aware of your change, seems I messed up in the conflict rebase :-(